source/locationHelper.py

```diff
@@ -1,18 +1,23 @@
 # A part of NonVisual Desktop Access (NVDA)
 # This file is covered by the GNU General Public License.
 # See the file COPYING for more details.
-# Copyright (C) 2017-2021 NV Access Limited, Babbage B.V.
+# Copyright (C) 2017-2024 NV Access Limited, Babbage B.V.
 
 """Classes and helper functions for working with rectangles and coordinates."""
 
-from collections import namedtuple
+from typing import NamedTuple
 import windowUtils
 import winUser
 from ctypes.wintypes import RECT, POINT, DWORD
 import wx
 
 
-class Point(namedtuple("Point", ("x", "y"))):
+class _Point(NamedTuple):
+	x: int
+	y: int
+
+
+class Point(_Point):
```

Comment:
Why not inherit from `NamedTuple` directly and have `x` and `y` on this class?

Reply:
This bug with python: https://stackoverflow.com/questions/61543768/super-in-a-typing-namedtuple-subclass-fails-in-python-3-8